lib/dynamoid/fields.rb

```diff
@@ -145,14 +145,14 @@ def update_attribute(attribute, value)
     #
     # @since 0.2.0
     def set_created_at
-      self.created_at = DateTime.now if Dynamoid::Config.timestamps
+      self.created_at = DateTime.now.in_time_zone(Time.zone) if Dynamoid::Config.timestamps
```

Comment:
DynamoDB is dependent on ActiveModel already.  I suppose this would add another part of rails as a dependency, so it is worth asking the question - do we want to do that.  It is generally expected that the normal use case for this gem is to integrate DynamoDB with a Rails project that uses ActiveRecord.  I suppose that may not be the use case for all users though!

Can you say more about your use case @rikas ?  Are you building a Rails-based app?